includes/admin/class-llms-admin-review.php

```diff
@@ -150,19 +165,20 @@ public function maybe_show_notice() {
 	/**
 	 * Round a number down to a big-ish round number
 	 *
-	 * @param   int $number input number
-	 * @return  int
-	 * @since   3.24.0
-	 * @version 3.24.0
+	 * @since 3.24.0
+	 * @since [version] Numbers less than 10 are not rounded & numbers less than 100 are rounded to the nearest 10.
+	 *
+	 * @param int $number Input number.
+	 * @return int
 	 */
 	public static function round_down( $number ) {
```

Comment:
what would you think about this golf thing?

```php
	public static function round_down( $number ) {

		if ( $number < 1 ) {
			return $number;
		}

		/**
		 *  $number < 10 => $dec === 0
		 *  9 < $number < 100 => $dec === 1
		 *  99 < $number < 1001 => $dec === 3
		 *  and son on...
		 */
		$dec = floor( log10( $number ) );
		/**
		 * so...
		 * if $dec is 0 => $pow is 1
		 * if $dec is 1 => $pow is 10
		 * if $dec is 2 => $pow is 100
		 * and so on...
		 */
		$pow = pow( 10, $dec );
		/**
		 * Any number > 10000 is rounded to 10000 (10^4).
		 * Then:
		 * 85 => 80;
		 * 100 => 100;
		 * 155 => 150;
		 * ...
		 */
		$number = $dec < 4 ? floor( $number / $pow ) * $pow : $pow;

		return $number;

	}
```

Reply:
Honestly... this looks completely unreadable to me. Mostly because I'm not familiar with these functions. I think my version is readable and performant enough.

Reply:
I see yes, it was just fun to me you know reduce that logic.